fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java

```diff
@@ -362,47 +355,45 @@ public static JSONObject getDefaultParams() throws Exception {
     public static Header[] getDefaultParamsAsHeaders(Header[] pHeaders) throws Exception {
         ArrayList<Header> headers = new ArrayList<Header>();
         JSONObject defaultParams = FH.getDefaultParams();
-        Iterator<String> it = defaultParams.keys();
-        while (it.hasNext()) {
+
+        for (Iterator<String> it = defaultParams.keys(); it.hasNext(); ) {
```

Comment:
The call to `next()` comes inside the loop so that its return value can be captured.